config.json

```diff
@@ -681,7 +681,11 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "algorithms",
+        "searching",
+        "lists",
+        "control-flow-(conditionals)",
```

Comment:
We've switched to `snake_case` naming convention for topics to increase consistency among tracks, so could you please choose them from the following list - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt

I'll create issues for exercises that have such inconsistency shortly